examples/addsvc/cmd/addsvc/pact_test.go

```diff
@@ -16,7 +16,6 @@ func TestPactStringsvcUppercase(t *testing.T) {
 	}
 
 	pact := dsl.Pact{
-		Port:     6666,
```

Comment:
How is this specified, now?

Reply:
not sure, examples didn't show anything similar.. I wanted to test it on CI as it seemed too much hassle to do locally... documentation is lacking w.r.t. the changes or it's buried somewhere deep :(

Reply:
@peterbourgon I see none of our CI environments actually runs it... do you have a local setup to test against?